keg_auth/model/__init__.py

```diff
@@ -75,7 +76,24 @@ class UserMixin(object):
     # is_active defines the complexities of how to determine what users are active. For instance,
     #   if email is in scope, we need to have an additional flag to verify users, and that would
     #   get included in is_active logic.
-    is_active = is_enabled
+    @hybrid_property
+    def is_active(self):
+        is_disabled = self.disable_date is not None and self.disable_date < arrow.utcnow()
```

Comment:
A few notes here:
- we end up duplicating logic by having this directly in `is_active`
  - we should have a separate hybrid property for `is_disabled_by_date`. That will:
    - encapsulate the logic for the case to be used in this and other mixins
    - provide a more descriptive name for the logic happening, which is much more specific than the name `is_disabled` suggests. Having `is_disabled` and `is_enabled` at the same naming specificity suggests a more complete overlap in use than is present here.
- `is_disabled_by_date` should be true when `disable_date` is provided and is present or past (i.e. <= instead of <)